src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/LibraryImportGenerator.csproj

```diff
@@ -12,6 +12,7 @@
     <!-- Disable RS2008: Enable analyzer release tracking
          Diagnostics in runtime use a different mechanism (docs/project/list-of-diagnostics.md) -->
     <NoWarn>RS2008;$(NoWarn)</NoWarn>
+    <AnalyzerLanguage>cs</AnalyzerLanguage>
```

Comment:
We do have at least one analyzer for vb, so we should probably set this to an empty string or `.` to make sure it goes in the right place in the eventual package.

We can also move the `CustomTypeMarshallerAnalyzer` to a different assembly if we want to keep this assembly packaged for C#-only.

Reply:
Ah, right. I'll remove it, since libraries infrastructure documents that it defaults to language agnostic.
```suggestion
```

Reply:
> We can also move the CustomTypeMarshallerAnalyzer to a different assembly if we want to keep this assembly packaged for C#-only.

I'm not sure if that is worth the extra packaging/dependencies. If it was the only analyzer/fixer, I think it would make more sense, but we also have the converting one.